### PR TITLE
KAN 2 MURALI PREFERED TIMESLOT

### PR DESCRIPTION
**Pull request by Murali03M**
1) PreferedTImeSlot component will return selected date and time.

Demo 

https://github.com/AYS-At-Your-Service/AYS/assets/73110751/530c38c6-2d3a-4f12-96a7-821dc9766334

### CODE REVIEW

File: Frontend/components/Prefered-timeslot.jsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    <div className="modal-container">
+      <div className="modal">
+        <div className="modal-cross-outer">
+          <div className="modal-cross" onClick={onClose}>
+            X
+          </div>
+        </div>
+        <div className="content-container">
+          <div>
+          <h3>Get a serivce later</h3>
```
Comment (aravind-digitech) [inline Frontend/components/Prefered-timeslot.jsx:49]: pls save all constants (strings, numbers) in a separate preferred-timeslot.consts.js file and import it for use

File: Frontend/components/Prefered-timeslot.jsx
Comment (aravind-digitech) [inline Frontend/components/Prefered-timeslot.jsx:1]: pls create a folder called preferredTimeSlot under components and move this and the css file there

File: Frontend/src/App.jsx
Comment (aravind-digitech) [inline Frontend/src/App.jsx:1]: pls revert all changes in this file

**Review by aravind-digitech — CHANGES_REQUESTED** [review]

File: Frontend/components/Prefered-timeslot.jsx (hunk 1 above)
Comment (Murali03M) [reply Frontend/components/Prefered-timeslot.jsx:49]: created the new file and constants are exported from there

File: Frontend/components/Prefered-timeslot.jsx
Comment (Murali03M) [reply Frontend/components/Prefered-timeslot.jsx:1]: folder was created

File: Frontend/src/App.jsx
Comment (Murali03M) [reply Frontend/src/App.jsx:1]: changes are reverted

**Review by aravind-digitech — APPROVED** [review]

**Review by GREEDnim — APPROVED** [review]

**Pull request merged**